src/utils/helpers.ts

```diff
@@ -29,6 +29,43 @@ export const getL2Wallet = (privateKey: string, l2Provider: Provider, l1Provider
   return new Wallet(privateKey, l2Provider, l1Provider);
 };
 
-export const executeCommand = (command: string) => {
-  execSync(`${command}`, { stdio: "inherit" });
+interface ExecuteOptions {
+  silent?: boolean;
+}
+export const executeCommand = (command: string, options: ExecuteOptions = {}): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    const [cmd, ...args] = command.split(" ");
+
+    const child = spawn(cmd, args, { stdio: options.silent ? "pipe" : "inherit" });
```

Comment:
Should we log out any commands prior to execution? I think it's necessary to be very transparent what's being run on the user's machine/terminal, so this would probably be `Logger.info(...)` and happen whether or not `options.silent` is true.

So as to avoid distracting the user with too much text, I'm fine with having this information be logged in `chalk.grey`.

Reply:
I feel like that would be just a lof of spam, even if it's just gray color.
I don't think there are any "dangerous" commands and in the future if anyone creates malicious custom module they would still be able to do stuff on your machine silently.